WorkflowTesting/Sources/WorkflowActionTester.swift

```diff
@@ -17,9 +17,9 @@
 import Workflow
 import XCTest
 
-extension WorkflowAction {
+public extension WorkflowAction {
```

Comment:
These access control declarations updates were performed by SwiftFormat.

Reply:
Yeah, we haven't pinned SwiftFormat to an exact version. I guess the newer version's doing this automatically.
I'll fix.

Reply:
Fixed [here](https://github.com/square/workflow-swift/pull/93), @nononoah  can you rebase after my PR lands?

Reply:
Happy to do so!